lib/libzpool/kernel.c

```diff
@@ -32,7 +32,6 @@
 #include <string.h>
 #include <zlib.h>
 #include <libgen.h>
-#include <sys/signal.h>
```

Comment:
Actually, I believe we can then remove `./lib/libspl/include/sys/signal.h` as well.